hack/build_channel_catalog.sh

```diff
@@ -45,49 +41,17 @@ done
 temp_dir=$(mktemp -d --suffix "-$(basename "$0")")
 [[ "$DELETE_TEMP_DIR" == "true" ]] && trap 'rm -rf $temp_dir' EXIT
 
-# Check we have the needed commands
-# opm
-opm_local_executable=$(which opm || true)
-if [[ "$opm_local_executable" ]]; then
-    opm_local_version=$(opm version | sed 's/.*OpmVersion:"//;s/".*//')
-fi
-
-if [[ -n "$opm_local_executable" && "$opm_local_version" == "$OPM_VERSION" ]]; then
-    log "Using local opm version $opm_local_executable"
-else
-    opm_download_url="https://github.com/operator-framework/operator-registry/releases/download/$OPM_VERSION/${GOOS}-${GOARCH}-opm"
-    log "Downloading opm from $opm_download_url to $temp_dir/opm"
-    curl -s -L "$opm_download_url" -o "$temp_dir/opm"
-    chmod u+x "$temp_dir/opm"
-    opm_local_executable="$temp_dir/opm"
-fi
+repo_root=$(git rev-parse --show-toplevel)
+$repo_root/boilerplate/openshift/golang-osd-operator/ensure.sh opm
+opm_local_executable="$repo_root/.opm/bin/opm"
 
-# grpcurl
-grpcurl_local_executable=$(which grpcurl || true)
-if [[ "$grpcurl_local_executable" ]]; then
-    grpcurl_local_version=$(grpcurl -version 2>&1 | cut -d " " -f 2)
-fi
-
-if [[ -n "$grpcurl_local_executable" && "$grpcurl_local_version" == "$GRPCURL_VERSION" ]]; then
-    log "Using local grpcurl version $grpcurl_local_executable"
-else
-    # mappings from https://github.com/fullstorydev/grpcurl/blob/master/.goreleaser.yml
-    os=$GOOS
-    arch=$GOARCH
-    [[ "$GOOS" == "darwin" ]] && os=osx
-    [[ "$GOARCH" == "386" ]] && arch=x86_32
-    [[ "$GOARCH" == "amd64" ]] && arch=x86_64
-    grpcurl_download_url="https://github.com/fullstorydev/grpcurl/releases/download/v$GRPCURL_VERSION/grpcurl_${GRPCURL_VERSION}_${os}_${arch}.tar.gz"
-    log "Downloading opm from $grpcurl_download_url to $temp_dir/grpcurl"
-    curl -s -L "$grpcurl_download_url" | tar -xzf - -C "$temp_dir" grpcurl
-    chmod u+x "$temp_dir/grpcurl"
-    grpcurl_local_executable="$temp_dir/grpcurl"
-fi
+$repo_root/boilerplate/openshift/golang-osd-operator/ensure.sh grpcurl
+grpcurl_local_executable="$repo_root/.grpcurl/bin/grpcurl"
 
 # ./hack/generate-operator-bundle-contents.py
-HERE=${0%/*}
-if [[ ! -x "$HERE/generate-operator-bundle-contents.py" ]]; then
-    echo "$HERE/generate-operator-bundle-contents.py is either missing or non-executable"
+bundle_contents_cmd="$repo_root/hack/generate-operator-bundle-contents.py"
+if [[ ! -x "$bundle_contents_cmd" ]]; then
+    echo "$bundle_contents_cmd is either missing or non-executable"
```

Comment:
```suggestion
    log "$bundle_contents_cmd is either missing or non-executable"
```
?

Reply:
For sure, I don't know what I was thinking 🤦